website/src/components/validation/validation.test.ts

```diff
@@ -37,18 +37,43 @@ test("Running Validation Tests for VALIDATE_free_form", () => {
   ).toBe(false);
 });
 
+//This function is here to avoid hard coded years in the tests, by returning the current year plus a specified number of years as a string
+function thisYearPlus(years: number): string {
+  let year = new Date().getFullYear();
+  year += years;
+  return year.toString();
+}
```

Comment:
> @theTisch21 for the test cases, you need to fix the graduation year. Otherwise, the tests will start to fail a year from now lol. You can do this by adding an optional parameter `currentYear` to the validation and passing it in the test cases but not on the prod code.
@daniel-sudz 
I'm not sure what needs fixing. This function (Lines 41-45 in the tests file) is designed to make the tests work years from now, and in the validation file, it takes the current year and compares the submitted year to that (Lines 38-51)

Reply:
Oh yeah nevermind I forgot that you switched it to be relative with ```thisYearPlus``` it should be good forever then pretty much